### DISCUSSION TITLE
Add the ability to add custom anchor link names when using the TOC widget

### DISCUSSION THREAD

**Discussion by joemalouff-MERGE** [elementor/elementor#29441] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

in the recent 3.25 update anchor links are now showing up more in search results. Although this is not inherently a bad thing, The TOC widget anchor names currently lack meaningful context. (For example: #elementor-toc__heading-anchor-2)

### Describe the solution you'd like.

Add the ability to customize the anchor links or pull dynamically from the header title. 

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by m1ga** [comment]
please join one of the other threads:
https://github.com/orgs/elementor/discussions/29144
https://github.com/orgs/elementor/discussions/24897
as they have some upvotes already

**Comment by joemalouff-MERGE** [comment]
Thank you!

On Wed, Nov 27, 2024 at 3:19 PM Michael Gangolf ***@***.***>
wrote:

> please join one of the other threads:
> https://github.com/orgs/elementor/discussions/29144
> https://github.com/orgs/elementor/discussions/24897
> as they have some upvotes already
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/elementor/discussions/29441#discussioncomment-11399731>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BNI5ODEL7YBNDTLLRR345GD2CZANXAVCNFSM6AAAAABSTXKXHSVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMZZHE3TGMI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

*Joe Malouff*

Director, Technology

*MERGE*

mergeworld.com
